test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -228,29 +228,28 @@ def create_driver(conf = APPLICATION_DEFAULT_CONFIG,
                     tag = 'test',
                     multi_tags = false)
     conf += USE_GRPC_CONFIG
+    use_secure_channel = !conf.include?('use_secure_channel false')
```

Comment:
Isn't this stale now?

Reply:
Right. Removed this and all other occurrences.